src/js/components/search/filters/awardID/AwardIDSearch.jsx

```diff
@@ -15,6 +15,28 @@ const propTypes = {
 };
 
 export default class AwardIDSearch extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            awardID: ''
+        };
+
+        this.changedInput = this.changedInput.bind(this);
+        this.applyAwardID = this.applyAwardID.bind(this);
+    }
+
+    changedInput(e) {
+        this.setState({
+            awardID: e.target.value
+        });
+    }
+
+    applyAwardID(e) {
+        e.preventDefault();
+        this.props.toggleAwardID(this.state.awardID);
```

Comment:
Reset `awardID`in the state after the filter is applied to clear the input value